portal-frontend/src/app/features/notice-of-intents/edit-submission/additional-information/roso/delete-structure-confirmation-dialog/delete-structure-confirmation-dialog.component.html

```diff
@@ -0,0 +1,14 @@
+<div mat-dialog-title>
+  <h4>Delete Structure Type</h4>
+</div>
+
+<div mat-dialog-content class="content">
+  <app-warning-banner>
+    <b>Warning:</b> Deleting the structure type can remove some content already saved to this page. <br />
+    Do you want to continue?
+  </app-warning-banner>
+  <div class="controls">
+    <button mat-stroked-button color="primary" (click)="onCancel()">cancel</button>
```

Comment:
You can replace onCancel and onConfirm with [mat-dialog-close]="false" and [mat-dialog-close]="true" to achieve the same results.

Reply:
done